Cargo.toml

```diff
@@ -12,6 +12,7 @@ keywords = ["rocket", "gzip", "brotli", "compression"]
 rocket = "0.5.0-rc.2"
 lazy_static = "1.4.0"
 futures = "0.3.17"
+tokio = { version = "1.20.0" }
```

Comment:
Probably want to match up with the version of tokio used by `rocket` itself: https://github.com/SergioBenitez/Rocket/blob/v0.5-rc/core/http/Cargo.toml#L36

Or maybe there's even a way to reference it from rocket, sometimes you can do `use rocket::tokio::whatever` but might not work in this case